views/update/confirm-overseas-entity-details.html

```diff
@@ -13,6 +13,12 @@
 
 {% set displayRegistrationDate = dateMacros.formatDate(registrationDate["day"], registrationDate["month"], registrationDate["year"]) %}
 
+{% if (journey == "remove") %}
+  {% set chooseLink = "/update-an-overseas-entity/overseas-entity-query?journey=remove" %}  
```

Comment:
you should be able to use OE_CONFIGS.JOURNEY_REMOVE_QUERY_PARAM instead of the ?journey=remove
as it is already inside a {% %} block i think you could use 
{% set chooseLink = "/update-an-overseas-entity/overseas-entity-query" = OE_CONFIGS.JOURNEY_REMOVE_QUERY_PARAM %}

Reply:
using var